src/app/window-handler.ts

```diff
@@ -1119,7 +1119,7 @@ export class WindowHandler {
       opts.alwaysOnTop = true;
     }
 
-    if (isWindowsOS && selectedParentWindow) {
+    if ((isMac || isWindowsOS) && selectedParentWindow) {
```

Comment:
as we just support Windows and macOS, you can just remove OS conditions
```suggestion
    if (selectedParentWindow) {
```

Reply:
Done!